components/stream-provider/package.json

```diff
@@ -66,10 +68,10 @@
     "testResultsProcessor": "jest-sonar-reporter"
   },
   "dependencies": {
+    "@browserd/shared": "^1.0.0",
```

Comment:
we switch to lerna-bootstrap style derivation, rather than npm file linking

Reply:
huh. I was so close! I just writing shared and using the lerna add command but never added the @browserd/ prefix. I know now!

Reply:
that's actually only necessary because i also changed the name of shared to use [this `@<thing>/<name>` pattern](https://docs.npmjs.com/about-scopes).